### DISCUSSION TITLE
Badges

### DISCUSSION THREAD

**Discussion by Sagar-Sharma-7** [community/community#38485] [Other Feature Feedback, Questions, & Ideas]
How can I get more achievements badges?

**Comment by david-engelmann** [comment]
[Here is a breakdown of all the achievements](https://github.com/Schweinepriester/github-profile-achievements). Best of luck!

**Comment by alimttth** [comment]
as for unlimited links, you can use the default README on your landing page to include all of your links, you can even make it beautiful by making a link icon instead of the actual link  this is what it looks like

**Comment by SilasRodrigues19** [comment]
@Sagar-Sharma-7 accessing [this link](https://github.com/drknzz/GitHub-Achievements) you will find a complete list of all achievements and their tiers and how to get each one
